applications/StructuralMechanicsApplication/custom_strategies/custom_schemes/explicit_central_differences_scheme.hpp

```diff
@@ -548,8 +579,12 @@ class ExplicitCentralDifferencesScheme
     /**
      * @brief This method performs some custom operations to initialize the scheme
      * @param rModelPart The model of the problem to solve
+     * @param DoF The current dimention of the problem
      */
-    virtual void SchemeCustomInitialization(ModelPart& rModelPart)
+    virtual void SchemeCustomInitialization(
+        ModelPart& rModelPart,
+        const SizeType DoF = 3
```

Comment:
same

Reply:
This is the most efficient, it is explicit, you need to avoid access and things in general

Reply:
bad name